fs/expose/dokan/__init__.py

```diff
@@ -731,36 +784,66 @@ def LockFile(self, path, offset, length, info):
             except KeyError:
                 locks = self._active_locks[path] = []
             else:
-                errno = self._check_lock(path, offset, length, None, locks)
-                if errno:
-                    return errno
+                status = self._check_lock(path, offset, length, None, locks)
+                if status:
+                    return status
             locks.append((info.contents.Context, offset, end))
-            return 0
+            return STATUS_SUCCESS
 
     @timeout_protect
     @handle_fs_errors
     def UnlockFile(self, path, offset, length, info):
-        end = offset + length
         with self._files_lock:
             try:
                 locks = self._active_locks[path]
             except KeyError:
-                return -ERROR_NOT_LOCKED
+                return STATUS_NOT_LOCKED
             todel = []
             for i, (lh, lstart, lend) in enumerate(locks):
                 if info.contents.Context == lh:
                     if lstart == offset:
                         if lend == offset + length:
                             todel.append(i)
             if not todel:
-                return -ERROR_NOT_LOCKED
+                return STATUS_NOT_LOCKED
             for i in reversed(todel):
                 del locks[i]
-            return 0
+            return STATUS_SUCCESS
+
+    @handle_fs_errors
+    def GetFileSecurity(self, path, securityinformation, securitydescriptor, securitydescriptorlength, neededlength, info):
+        securitydescriptor = ctypes.cast(securitydescriptor, libdokan.PSECURITY_DESCRIPTOR)
+        path = self._dokanpath2pyfs(path)
+        if self.fs.isdir(path):
+            res = libdokan.GetFileSecurity(
+                os.path.expanduser('~'),
```

Comment:
I added the param but I don't get how to give acces to user config 
https://github.com/PyFilesystem/pyfilesystem/pull/258/commits/1a43556d85251b7caac8a54aa9c24f04adb58dee#diff-bcd6666601556e9c56d197740debee23R312